jib-gradle-plugin/src/integration-test/java/com/google/cloud/tools/jib/gradle/JibPluginIntegrationTest.java

```diff
@@ -190,6 +190,11 @@ public void testBuild_empty() throws IOException, InterruptedException {
         new Command("docker", "inspect", "-f", "{{.Created}}", targetImage).run().trim());
   }
 
+  @Test
+  public void testBuild_multipleTags() throws IOException, InterruptedException {
+    // TODO: Implement
```

Comment:
If this is to be implemented in a later PR, I'd remove it from here. It's just a test method, and I don't think we need a placeholder.

Reply:
Oh yea, I will implement this in this PR. I wanted to solicit some comments about the way `BaseImageParameters` and `TargetImageParameters` are laid out first.